libmscore/mscore.h

```diff
@@ -188,13 +188,18 @@ const int STAFF_GROUP_MAX = int(StaffGroup::TAB) + 1;      // out of enum to avo
 
 enum class BarLineType {
       NORMAL           = 1,
+      SINGLE           = NORMAL,
       DOUBLE           = 2,
       START_REPEAT     = 4,
       END_REPEAT       = 8,
       BROKEN           = 0x10,
+      DASHED           = BROKEN,
```

Comment:
Dashed is back? What does that means if it's the same as Broken?

Reply:
It is just a different name for the same thing